src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -408,7 +408,7 @@ export const getFormSchema = ({
                       volumeSize: {
                         title: "Volume size (GB)",
                         type: "number" as "number",
-                        default: 500,
```

Comment:
Would prefer to update this constant and then use that variable in place of this magic number! https://github.com/evergreen-ci/spruce/blob/de2fac705d2400ea62c68708a3ee8ebbc4b18a98/src/components/Spawn/spawnHostModal/constants.ts#L1

Reply:
perf thanks!